api/config/config_prod.json

```diff
@@ -3,6 +3,8 @@
     "baseUrl": "https:\/\/api.firecloud.org",
     "clusterMaxAgeDays": 14,
     "clusterIdleMaxAgeDays": 7,
+    "clusterDefaultMachineType": "n1-standard-2",
```

Comment:
swapped Prod and Perf

Reply:
:man_facepalming:  Great catch.

That would have rendered this PR pretty useless.